doc/dev/installation.rst

```diff
@@ -81,6 +81,21 @@ The essential steps are:
 
    .. group-tab:: Linux Debian/Ubuntu
 
+      .. warning::
+
+        If using Anaconda or Miniconda, please make sure to upgrade ``libstdcxx-ng``:
+
+        .. code-block:: console
+
+          conda install -c conda-forge libstdcxx-ng
+
+        If not, you will receive the following error when running ``make test`` because the conda
+        environment is using old versions of ``libstdcxx-ng``.
+
+        .. code-block:: console
+
+          'GLIBCXX_3.4.30' not found
```

Comment:
```suggestion
          'GLIBCXX_3.4.x' not found
```

The version will be OS dependent, so we can likely just replace `40` with `x` --- on my machine it is 3.4.30, and I suspect Ubuntu 20.04 it will be 3.4.20